docs/devices/TV02-Zigbee.md

```diff
@@ -20,7 +20,7 @@ pageClass: device-page
 | Description | Thermostat radiator valve |
 | Exposes | battery_low, lock (state), open_window, open_window_temperature, comfort_temperature, eco_temperature, climate (preset, local_temperature_calibration, local_temperature, current_heating_setpoint, system_mode), heating_stop, frost_protection, boost_timeset_countdown, holiday_temperature, holiday_start_stop, working_day, schedule, schedule_monday, schedule_tuesday, schedule_wednesday, schedule_thursday, schedule_friday, schedule_saturday, schedule_sunday, online, error_status, linkquality |
 | Picture | ![TuYa TV02-Zigbee](https://www.zigbee2mqtt.io/images/devices/TV02-Zigbee.jpg) |
-| White-label | Moes TV01-ZB, AVATTO TRV06, Tesla Smart TSL-TRV-TV01ZG, Unknown/id3.pl GTZ08 |
+| White-label | AVATTO TRV06, Moes TV01-ZB, PRIOS TV02, Tesla Smart TSL-TRV-TV01ZG, Unknown/id3.pl GTZ08 |
```

Comment:
This part is generated, please update https://github.com/Koenkk/zigbee-herdsman-converters/blob/2ec4b6fcdf6c808b6b028a541d658492c9ffe6a7/src/devices/tuya.ts#L2658 instead.